enforcer/src/main/resources/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM adoptopenjdk/openjdk11:jre-11.0.11_9-alpine
+FROM adoptopenjdk/openjdk11:jdk-11.0.11_9-alpine
```

Comment:
Why do we switch to jdk here?

Reply:
this is introduced from the main branch. So let's keep it as it is, until the main branch is fixed.